packages/file_selector/file_selector/lib/file_selector.dart

```diff
@@ -9,7 +9,23 @@ import 'package:file_selector_platform_interface/file_selector_platform_interfac
 export 'package:file_selector_platform_interface/file_selector_platform_interface.dart'
     show XFile, XTypeGroup;
 
-/// Open file dialog for loading files and return a file path
+/// Opens a file selection dialog and returns the path chosen by the user.
+///
+/// [acceptedTypeGroups] is a list of file type groups that can be selected in
+/// the dialog. How this is displayed depends on the pltaform, for example:
+/// - On Windows and Linux, each group will be an entry in a list of filter
+///   options.
+/// - On macOS, the union of all types allowed by all of the groups will be
+///   allowed.
+///
+/// [initialDirectory] is the full path to the directory that will be displayed
+/// when the dialog is opened. When not provided, the platform will pick an
+/// initial location.
+///
+/// [confirmButtonText] is the text in the confirmation button of the dialog.
+/// When not provided, the default OS label is used (for example, "Open").
+///
+/// Returns `null` if the user cancels the operation.
```

Comment:
```suggestion
/// Returns `null` if the user cancels the operation. Except on the Web platform.
/// @see https://github.com/flutter/flutter/issues/92176
```

Reply:
I think since this is a bug, we should leave it out; it's very likely to become stale (since the fix will be in a separate package).